.github/workflows/field-planner.yaml

```
@@ -0,0 +1,97 @@
+            VERSION=${GITHUB_REF#refs/tags/}  # Use tag name for version
+          elif [[ "${GITHUB_REF}" == refs/heads/* ]]; then
+            # Use branch name as version, replacing '/' with '-'
+            VERSION=${GITHUB_REF#refs/heads/}
+            VERSION=$(echo "${VERSION}" | sed 's|/|-|g')
+          else
+            VERSION="latest"  # Default version if not a tag or branch
+          fi
+          echo "VERSION=$VERSION" >> $GITHUB_ENV
+
+      - name: Git commit_hash for service
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:2:36: Double quote to prevent globbing and word splitting

(shellcheck)